spec/functions/module_metadata/os_supported_spec.rb

```diff
@@ -14,87 +14,93 @@
 
     let(:facts) { facts }
 
-    let(:module_metadata){{
-      'name' => 'test-module',
-      'version' => '0.0.1',
-      'operatingsystem_support' => [
-        {
-          'operatingsystem'        => 'Aardvark',
-          'operatingsystemrelease' => [ '14.999' ]
-        },
-        {
-          'operatingsystem'        => 'Ubuntu',
-          'operatingsystemrelease' => [ '14.999' ]
-        },
-        {
-          'operatingsystem'        => 'Banana',
-          'operatingsystemrelease' => [ '1.2', '2.3' ]
-        }
-      ]
-    }}
+    let(:module_metadata) do
+      {
+        'name' => 'test-module',
+     'version' => '0.0.1',
+     'operatingsystem_support' => [
+       {
+         'operatingsystem'        => 'Aardvark',
+         'operatingsystemrelease' => [ '14.999' ]
+       },
+       {
+         'operatingsystem'        => 'Ubuntu',
+         'operatingsystemrelease' => [ '14.999' ]
+       },
+       {
+         'operatingsystem'        => 'Banana',
+         'operatingsystemrelease' => [ '1.2', '2.3' ]
+       },
+     ]
+      }
+    end
 
     context 'with no version matching' do
       context 'with a match' do
-        it 'should return true' do
+        it 'returns true' do
           is_expected.to run.with_params(module_metadata).and_return(true)
         end
       end
 
       context 'without a match' do
-        let(:module_metadata){{
-          'name' => 'test-module',
-          'version' => '0.0.1',
-          'operatingsystem_support' => [
-            {
-              'operatingsystem'        => 'Aardvark',
-              'operatingsystemrelease' => [ '14.999' ]
-            },
-            {
-              'operatingsystem'        => 'Banana',
-              'operatingsystemrelease' => [ '15.123' ]
-            }
-          ]
-        }}
+        let(:module_metadata) do
```

Comment:
Is the indentation for name, and the spacing of the closing braces & brackets good here?

Reply:
> Is the indentation for name, and the spacing of the closing braces & brackets good here?

Nope, I missed that.  Thanks.